src/key.rs

```diff
@@ -2,7 +2,7 @@ use std::io;
 
 use ff::{PrimeField, PrimeFieldBits};
 use group::Curve;
-use rand_core::{CryptoRng, RngCore};
```

Comment:
Because `CryptoRng` is now a supertrait of `RngCore`, we should be able to remove the redundant `RngCore` import here and in `PrivateKey::generate`'s trait bounds.

https://github.com/filecoin-project/bls-signatures/blob/a98545fdff617dfe839245804cfcfdfddc294cd5/src/key.rs#L76

Reply:
I removed it in a new commit